tools/schema/seasonalpass_missions.json

```diff
@@ -148,29 +165,155 @@
+									"type": "string",
+									"anyOf": [
+										{
+											"title": "Custom",
+											"description": "In case content is missing from the dropdowns."
+										},
+										{
+											"title": "Dungeon",
+											"description": "Content that is considered a dungeon.",
+											"enum": [
+												"labs",
+												"white",
+												"orange",
+												"magenta",
+												"lightblue",
+												"yellow",
+												"reverie",
+												"willows",
+												"corridors",
+												"lime",
+												"pink",
+												"gray",
+												"lightgray",
+												"cyan",
+												"purple",
+												"teal",
+												"shiftingcity",
+												"forum",
+												"blue",
+												"brown",
+												"skt",
+												"hexfallruten",
+												"hexfallhycenea",
+												"depths",
+												"zenith"
+											]
+										},
+										{
+											"title": "Boss",
+											"description": "Content that is considered a boss.",
+											"enum": [
+												"kaul",
+												"azacor",
+												"snowspirit",
+												"horseman",
+												"eldrask",
+												"hekawt",
+												"godspore",
+												"sirius"
+											]
+										},
+										{
+											"title": "Strike",
+											"description": "Content that is considered a strike.",
+											"enum": [
+												"verdant",
+												"sanctum",
+												"mist",
+												"remorse",
+												"portal",
+												"ruin"
+											]
+										},
+										{
+											"title": "POI",
+											"description": "To be used only with 'poi_biomes'.",
+											"enum": [
+												"wolfswood",
+												"keep",
+												"starpoint"
+											]
+										},
+										{
+											"title": "Other",
+											"description": "Content that doesn't fit in the other categories.",
+											"enum": [
+												"corridorsroom",
+												"arena",
+												"rush",
+												"rushwave",
+												"gallery",
+												"galleryround",
+												"r1daily",
+												"r2daily",
+												"r3daily",
+												"delvebounty",
+												"fishingcombat"
+											]
+										}
+									]
+								}
+							},
+							{
+								"title": "All Wool Dungeons",
+								"description": "For 'content': Automatically sets all wool dungeons.",
+								"type": "array",
+								"enum": [
+									[
+										"white",
+										"orange",
+										"magenta",
+										"lightblue",
+										"yellow",
+										"lime",
+										"pink",
+										"gray",
+										"lightgray",
+										"cyan",
+										"purple",
+										"blue",
+										"brown"
+									]
+								]
+							},
+							{
+								"title": "All Bonus Dungeons",
+								"description": "For 'content': Automatically sets all bonus dungeons.",
+								"type": "array",
+								"enum": [
+									[
+										"labs",
```

Comment:
Labs should not be counted here unless it traditionally is (Which it shouldnt be)